site/public/css/polls.css

```diff
@@ -43,6 +43,11 @@
     vertical-align: top;
 }
 
+.radio {
```

Comment:
No, that's because the "no response" one doesn't use text indent here... I'm still trying figure out the best way to make it to and solve the problem with formatting the square checkbox buttons - which was more messed up originally when there are multiple lines.